packages/neuron-ui/src/components/NervosDAO/hooks.ts

```diff
@@ -24,14 +24,8 @@ import { ckbCore, getHeaderByNumber } from 'services/chain'
 import { isErrorWithI18n } from 'exceptions'
 import { calculateMaximumWithdraw } from '@nervosnetwork/ckb-sdk-utils'
 
-const {
-  MIN_AMOUNT,
-  MILLISECONDS_IN_YEAR,
-  MIN_DEPOSIT_AMOUNT,
-  MEDIUM_FEE_RATE,
-  SHANNON_CKB_RATIO,
-  MAX_DECIMAL_DIGITS,
-} = CONSTANTS
+const { MIN_AMOUNT, MILLISECONDS_IN_YEAR, MIN_DEPOSIT_AMOUNT, MEDIUM_FEE_RATE, SHANNON_CKB_RATIO, MAX_DECIMAL_DIGITS } =
```

Comment:
There is seem no change but for code style.

Reply:
resolved